sdk/tables/azure-tables/samples/javascript/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "azure-tables-samples-js",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Azure Tables client library samples for Javascript",
+  "engine": {
+    "node": ">=8.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure/azure-sdk-for-js.git"
+  },
+  "keywords": [
+    "Azure",
+    "Storage",
+    "Tables",
+    "Node.js"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js#readme",
+  "sideEffects": false,
+  "dependencies": {
```

Comment:
Should we require @azure/tables here as well?

Reply:
Yeah, I wasn't adding it because the package is not yet published. However it looks like we can add it here with a local reference. I'll do that 😄 once we publish we can start grabbing it from NPM